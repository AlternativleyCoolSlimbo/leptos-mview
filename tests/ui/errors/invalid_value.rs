use leptos_mview::component;

fn main() {
    component! {
        div a=a {}
    };
}