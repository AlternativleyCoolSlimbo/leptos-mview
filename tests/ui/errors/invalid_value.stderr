error: invalid value: expected bracket, block or literal
 --> tests/ui/errors/invalid_value.rs:5:15
  |
5 |         div a=a {}
  |               ^
